### PR TITLE
added WHFNP to readme

### PR DESCRIPTION
**Pull request by swentling87**
#71 Added Women Hack For Non-Profits

### CODE REVIEW

**Comment by cristianoliveira** [comment]
@harsh183 WDTY?

File: README.md [hunk 1]
```diff
@@ -239,6 +239,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     Lambda Ladies is an online community for anyone who identifies as a girl or woman and is interested in functional programming. Complete beginners are welcome.
     
+  - [Women Hack For Non-Profits (WHFNP)](http://www.womenhackfornonprofits.com/)
+
+    Women Hack for Non-Profits (WHFNP) is a volunteer community that enhances the skills of women in technology, supports their development and provides opportunities to explore open source. The group matches skilled women to open source projects that are being developed for non-profits, charities and individuals that have a positive social impact.
```
Comment (harsh183) [inline README.md]: I think 'charities and individuals that have a positive social impact.' can be changed to 'charities and individuals to create a positive social impact.'

The use of positive is a little redundant, but I'll let you decide on keeping it or not.

**Comment by swentling87** [comment]
Sounds good! I went ahead and updated that.

**Comment by harsh183** [comment]
Looks great. I'm merging.

**Pull request merged**